smoke-test.sh

```diff
@@ -266,8 +259,11 @@ function runWebTests() {
     doCommand C# razor "$@" new restore run
 
     doCommand F# web "$@" new restore run
-    doCommand F# mvc "$@" new restore run
-    doCommand F# webapi "$@" new restore run
+    # XXX workaround for F# web template issue
+    # non-empty F# web tests are not currently expected to pass because they rely on code that does not ship in the SDK
+    # doCommand F# mvc "$@" new restore run
+    # doCommand F# webapi "$@" new restore run
+    # XXX end workaround
```

Comment:
What does a commented out workaround section mean?

Reply:
Hopefully made the intent here more clear; we now emit a warning that we're skipping known-failing tests with a link to the issue.